### ISSUE TITLE
[auth] Wants to install libpam-cracklib, which doesn't exist in Bookworm and later

### ISSUE THREAD

**Issue by Alphix** [debops/debops#2304]
Just putting this here as a reminder (doesn't affect Debian stable, so not a real issue yet).

The `auth` role wants to install the [libpam-cracklib](https://packages.debian.org/bullseye/libpam-cracklib) package, which doesn't exist in Bookworm/Testing.

The successor in title appears to be [libpam-pwquality](https://packages.debian.org/bullseye/libpam-pwquality)

**Comment by stefangweichinger** [comment]
hits me as well

**Comment by KillerX** [comment]
I just encountered this on Bookworm which is stable now. Is there any known workaround?

**Comment by KillerX** [comment]
It would appear that this is actually fixed in master (https://github.com/debops/debops/commit/db1fb5d007feb51dd3a38d6b6c8ab0062452eeb4) but not released yet.

**Comment by Alphix** [comment]
> It would appear that this is actually fixed in master ([db1fb5d](https://github.com/debops/debops/commit/db1fb5d007feb51dd3a38d6b6c8ab0062452eeb4)) but not released yet.

Yeah, I'd consider it a partial fix though. On `bookworm` and later, I think the `auth` role should support using `libpam-pwquality` instead. I'll prepare a PR at some point...
